androidComposePrototype/src/main/java/com/genesys/cloud/messenger/androidcomposeprototype/ui/testbed/TestBedViewModel.kt

```diff
@@ -601,6 +634,24 @@ class TestBedViewModel :
             }
         return builder.build().toString()
     }
+
+    private fun generateNonce(): String = UUID.randomUUID().toString()
```

Comment:
That looks strange. Do not understand the need to generate nonce, pass it to buildImplicitAuthFlow etc. 
Why you decided to do that?